src/components/Table.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState, useEffect } from 'react';
+import './Table.css';
+
+export default function Table() {
+  const [items, setItems] = useState([]);
+
+  useEffect(() => {
+    getData();
+  });
+
+  async function getData() {
+    const curated = await fetch(
```

Comment:
Instead of making request here try making a utility file that will contain functions to make network request for each endpoint